R/generateEDGEdata.R

```diff
@@ -543,3 +543,41 @@ generateEDGEdata <- function(input_folder, output_folder, cache_folder = "cache"
   }
 
 }
+
+#' @export
+#' @rdname generateEDGEdata
+calcgenerateEDGEdata <- function(input_folder, output_folder,
```

Comment:
ah I see, that's the candidate I was missing in https://github.com/pik-piam/mrremind/pull/245/files